sw/device/tests/dif/dif_spi_device_unittest.cc

```diff
@@ -2,17 +2,16 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 
-#include <limits>
-
-extern "C" {
 #include "sw/device/lib/dif/dif_spi_device.h"
-#include "spi_device_regs.h"  // Generated.
-}  // extern "C"
+
+#include <limits>
```

Comment:
This should also be `#include <limits.h>` which suggests it's not used for anything.